diff --git a/runtime/lua/vim/_defaults.lua b/runtime/lua/vim/_defaults.lua
index 8e850f4cd3..c6e714b879 100644
--- a/runtime/lua/vim/_defaults.lua
+++ b/runtime/lua/vim/_defaults.lua
@@ -797,10 +797,4 @@ end
 
 --- Default options
 do
-  --- Default 'grepprg' to ripgrep if available.
-  if vim.fn.executable('rg') == 1 then
-    -- Use -uu to make ripgrep not check ignore files/skip dot-files
-    vim.o.grepprg = 'rg --vimgrep -uu '
-    vim.o.grepformat = '%f:%l:%c:%m'
-  end
 end
